lib/spawnGPG.js

```diff
@@ -96,14 +96,39 @@ module.exports.streaming = function(options, args, cb) {
   // Go for it
   var gpg = spawnIt(args, cb);
 
+  let stderrData = []
+  gpg.stderr.on('data', (data) => {
+    stderrData.push(data);
+  });
```

Comment:
I'll have to take a closer look at how this `streaming` func is used, but initially thoughts:

can you rename `stderrData` to `error`, rename `data` to `buf` and initialize `error` as an empty string and then change the push so that it appends to `error` so it is consistent with this:

https://github.com/drudge/node-gpg/blob/510c58c8af8f4a4b8a59c8c4873a76f21357adcf/lib/spawnGPG.js#L28-L38